sdk/go/azure/compute/orchestratedVirtualMachineScaleSet.go

```diff
@@ -71,7 +69,10 @@ type OrchestratedVirtualMachineScaleSet struct {
 	DataDisks               OrchestratedVirtualMachineScaleSetDataDiskArrayOutput           `pulumi:"dataDisks"`
 	EncryptionAtHostEnabled pulumi.BoolPtrOutput                                            `pulumi:"encryptionAtHostEnabled"`
 	EvictionPolicy          pulumi.StringPtrOutput                                          `pulumi:"evictionPolicy"`
-	Identity                OrchestratedVirtualMachineScaleSetIdentityPtrOutput             `pulumi:"identity"`
+	Extensions              OrchestratedVirtualMachineScaleSetExtensionArrayOutput          `pulumi:"extensions"`
```

Comment:
Same thing - related to a Pulumi addition?

Reply:
Yup!